acid/features/history/model.py

```diff
@@ -104,7 +104,13 @@ class ZuulBuild(db.Entity):
     @property
     def build_number(self):
         try:
-            return int(self.log_url.split('/')[-3])
+            split_url = self.log_url.split('/')
```

Comment:
Since this code involves some weird 'logic', Id suggest to separate this into a private method like: `def _get_build_number_from_log_url`. In this property just return the returned value from that method.
Inside the method itself, put all the comments explaining what it does, and why it does it, and also why it does it just like its being done here - splitting the url.

Side note, this seems like a configuration thing: which part of the url to take as a build number. Maybe consider putting this into config?

Reply:
I think separate to private method is the best solution for now